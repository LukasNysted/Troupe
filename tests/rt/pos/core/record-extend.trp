let val r = {} 
    val r2 = {r with b = 2}
    val field = "a"
    val value = 10 
    val r3 = recordExtend (r2, field, value)
    val field2 = "c" raisedTo `{alice}`
 in recordExtend (r3, field2, 55)
end