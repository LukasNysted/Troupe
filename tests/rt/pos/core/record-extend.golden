2024-03-11T10:25:40.461Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
>>> Main thread finished with value: {b=2@{}%{}, a=10@{}%{}, c=55@{alice}%{alice}}@{alice}%{alice}
